runner/src/server/lib/notifyService.ts

```diff
@@ -0,0 +1,58 @@
+import { NotifyClient } from "notifications-node-client/client/notification";
```

Comment:
`notifyService.js` still exists btw. Your IDE might be hiding it - for me webstorm thinks it's a compiled from `notifyService.ts`

Reply:
Yep I was missing it. Deleted. Thanks